main.cc

```diff
@@ -60,7 +73,17 @@ int main(int argc, char *argv[]) {
       }
       break;
     case smctemp::kOpReadCpuTemp:
-      std::cout << std::fixed << std::setprecision(1) << smc_temp.GetCpuTemp();
+      double temp = 0.0;
+      while (attempts > 0) {
+        temp = smc_temp.GetCpuTemp();
+        attempts--;
+        if (attempts > 0 && (temp == 0.0 || std::isnan(temp))) {
+          usleep(1000000);
```

Comment:
Let's add single quote to be easier to read.
```suggestion
          usleep(1'000'000);
```

Reply:
Personal preference but added it anyway